src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawTextWriterGenerator.ttinclude

```diff
@@ -759,8 +755,8 @@ namespace System.Xml
                             _stream = null;
                         }
                     }
-                }
-<# if (WriterType == RawTextWriterType.Encoded) { #>
+                }<#
+if (WriterType == RawTextWriterType.Encoded) { #>
```

Comment:
NIT: feels like spacing will not be right on generated files, but i might miss something, not gonna comment about spacing anymore